.github/workflows/aiida.yml

```diff
@@ -24,5 +23,7 @@ jobs:
       run: |
         pip install -e aiida_qe_basic
         conda install -c conda-forge jupyter papermill
+        pip install aiida-shell
```

Comment:
Waiting for conda-forge package to be released https://github.com/conda-forge/staged-recipes/pull/28108